source/geometries/GeometryBase.h

```diff
@@ -40,15 +40,18 @@ namespace nexus {
     /// Returns the 3 dimensions of the geometry (x, y, z)
     G4ThreeVector GetDimensions();
 
-    /// Returns true if geometry has a drift field
-    G4bool GetDrift() const;
```

Comment:
This was an obsolete function, which was not used practically in the code. Since this PR adds a displacement in the 3 dimensions (instead of only in `z`), I took the chance of restructuring the code using a generic 3D displacement. This was something I had already started in a previous PR, which I have closed now, since it is being implemented here.